mmcv/runner/fp16_utils.py

```diff
@@ -21,7 +23,7 @@
     pass
 
 
-def cast_tensor_type(inputs, src_type, dst_type):
+def cast_tensor_type(inputs, src_type: torch.dtype, dst_type: torch.dtype):
```

Comment:
We can set return type with any to avoid use `type: ignore`

Reply:
I tried it but can't fix it.